pydeepmerge/strategies/decorators/_decorators.py

```diff
@@ -0,0 +1,34 @@
+import functools as ft
+from typing import Mapping
+
+from pydeepmerge import deep_merge
+from pydeepmerge._deep_merge import MergeStrategy
+from pydeepmerge.types import Key
+
+
+def recurse(strategy: MergeStrategy) -> MergeStrategy:
```

Comment:
Can each of these functions be split into their own commits for atomicity's sake? It'll help with the changelog generation since each of these could arguably be described as a new feature into their own right.

note: if the commit starts with `new:`, `fix:`, or `chg:`, the commit messages will be appended to the correct subsection of the changelog when a new one gets generated. 

Added #4 to address the lack of contribution documentation that should've explained this

Reply:
Okay! After a bunch of rebasing (my first time!) it should be done!